samples/excel/85-preview-apis/data-types-entity-values.yaml

```diff
@@ -0,0 +1,603 @@
+          },
+          {
+            categoryID: 5,
+            categoryName: "Grains/Cereals",
+            description: "Breads, crackers, pasta, and cereal"
+          },
+          {
+            categoryID: 6,
+            categoryName: "Meat/Poultry",
+            description: "Prepared meats"
+          },
+          {
+            categoryID: 7,
+            categoryName: "Produce",
+            description: "Dried fruit and bean curd"
+          },
+          {
+            categoryID: 8,
+            categoryName: "Seafood",
+            description: "Seaweed and fish"
+          }
+        ];
+
+        const suppliers = [
+          {
+              "supplierID": 1,
+              "companyName": "Exotic Liquids",
+              "contactName": "Charlotte Cooper",
+              "contactTitle": "Purchasing Manager",
+          },
+          {
+              "supplierID": 2,
+              "companyName": "New Orleans Cajun Delights",
+              "contactName": "Shelley Burke",
+              "contactTitle": "Order Administrator",
+          },
+          {
+              "supplierID": 3,
+              "companyName": "Grandma Kelly's Homestead",
+              "contactName": "Regina Murphy",
+              "contactTitle": "Sales Representative",
+          },
+          {
+              "supplierID": 4,
+              "companyName": "Tokyo Traders",
+              "contactName": "Yoshi Nagase",
+              "contactTitle": "Marketing Manager",
+              "address": "9-8 Sekimai Musashino-shi",
+          },
+          {
+              "supplierID": 5,
+              "companyName": "Cooperativa de Quesos 'Las Cabras'",
+              "contactName": "Antonio del Valle Saavedra",
+              "contactTitle": "Export Administrator",
+          },
+          {
+              "supplierID": 6,
+              "companyName": "Mayumi's",
+              "contactName": "Mayumi Ohno",
+              "contactTitle": "Marketing Representative",
+          },
+          {
+              "supplierID": 7,
+              "companyName": "Pavlova, Ltd.",
+              "contactName": "Ian Devling",
+              "contactTitle": "Marketing Manager",
+          },
+          {
+              "supplierID": 8,
+              "companyName": "Specialty Biscuits, Ltd.",
+              "contactName": "Peter Wilson",
+              "contactTitle": "Sales Representative",
+          }
+        ];
+    language: typescript
+template:
+    content: |-
+        <section class="ms-font-m">
+          <p>This sample shows how to create entity value data types for each row in a table.</p>
+          <p>In particular, this sample shows the card layout options of an entity value, including the title, an image, collapsible sections, and nested entity values.</p>
+          <p>To see the entity value, click the icon to the left of the title in the <strong>Product</strong> column after selecting <strong>Add entity values</strong>.</p>
```

Comment:
Consider moving this line to after the **Add entity values** button. Then, the user reads it after they've clicked the button.